web/store/node.ts

```diff
@@ -65,7 +65,15 @@ export default function nodeStore() {
     },
 
     async apply(n: V1Node) {
-      await nodeAPI.applyNode(n);
+      if (n.metadata?.name) {
+        await nodeAPI.applyNode(n);
+      } else if (!n.metadata?.name && n.metadata?.generateName) {
```

Comment:
```suggestion
      if (n.metadata?.name) {
        await nodeAPI.applyNode(n);
      } else if (n.metadata?.generateName) {
```
This condition (`!n.metadata?.name`) will always be true?

Reply:
it will, I left it for clarity but I can remove it since it is redundant